docs/en/week04/practicum04.sbv

```diff
@@ -1293,29 +1294,29 @@ This is very nice of you. Let's keep
 going.
 
 0:34:32.360,0:34:36.320
-A♭, then we have an E♭, and then a A♭.
+A flat, then we have E flat and A flat.
```

Comment:
Nooooo, you're undoing my edits! Hahaha!

Reply:
Hahah! Sorry! I'll revert this and all the ones below